src/main/java/org/radarbase/appserver/controller/GithubEndpoint.java

```diff
@@ -48,7 +48,7 @@ public GithubEndpoint(GithubClient githubClient) {
     @Authorized(
             permission = AuthConfig.AuthPermissions.READ,
             entity = AuthConfig.AuthEntities.SUBJECT,
-            permissionOn = PermissionOn.PROJECT)
+            permissionOn = PermissionOn.SUBJECT)
```

Comment:
Same issue with subject too, you would need `subjectId` in the parameters which is indeed not required for github requests. I would just remove the `permissionOn` clause from `@Authorized`.

Reply:
Ah, my bad. I'll remove it.